container.go

```diff
@@ -121,15 +89,17 @@ func (container Container) Connect(val interface{}, id ...string) {
 				impl = idAndImpl[1]
 			}
 
-			comp.dependencies = append(comp.dependencies, dependency{
+			dependinces = append(dependinces, dependency{
 				id:    id,
 				name:  sf.Name,
 				index: i,
 				typ:   depRt,
 				impl:  impl,
 			})
+
 		} else if (sf.Type.Kind() == reflect.Ptr || sf.Type.Kind() == reflect.Interface) && rv.Field(i).IsNil() {
-			panic(tagMissingError{field: sf})
+			// Is this required
```

Comment:
Trying to use an external library where `wire:"-"` can't be added, so is more of a question :)

Reply:
This is for validation purpose, to ensure all dependency is properly added and connected.

Hmm, I never consider the case where external library might have nil pointer or interface.
How if we add another variant of `Connect` function that allows bypassing this check?